### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by veos-sxarr**
We fixed bugs we found during test.
We also updated make files and spec files to create aveo package for VH and aveorun for VE in order to create debuginfo packages.

### CODE REVIEW

File: doc/Restriction.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Restriction
 
-VEO does not support:
- - to use quadruple precision real number a variable length character string 
-   as a return value and an argument of Fortran subroutines and functions,
+VEO does not support to execute VEO program without setting the number of HugePages more than 32.
+VEO does not support to use quadruple precision real number a variable length character string as a return value and an argument of Fortran subroutines and functions,
```
Comment (efocht) [inline doc/Restriction.md]: fixup language a bit later

File: src/Makefile [hunk 2]
```diff
@@ -107,25 +102,23 @@ $(BLIB)/libveo.a: $(VHLIB_OBJS) | $$(@D)/
...
-		-L$(BLIB) -laveoVE -lveio -lpthread
+	$(NFORT) $(NFORTFLAGS) -ftrace -cxxlib -fopenmp -o $@ $^ \
+		$(NLDFLAGS) -laveoVE -lveio -lpthread
 
 $(BLIBEX)/relink_aveorun: ../scripts/relink_aveorun.in
 	sed -e "s,@libexecdir@,$(DEST)/libexec,g" -e "s,@velibdir@,$(BUILD)/lib,g" \
 		< $< > $@
 	chmod 755 $@
 
+$(BLIBEX)/mk_veorun_static: ../scripts/relink_aveorun.in
```
Comment (efocht) [inline src/Makefile]: Do we need twice the same script? IMO relink_aveorun is a more appropriate name and we should keep it and symlink the old name for compatibility.

File: doc/GettingStarted.md.in [hunk 3]
```diff
@@ -3,60 +3,60 @@ VE Offloading framework (VEO) is a framework to provide accelerator-style
...
+You need not uninstall veoffload and veoffload-veorun which are the
+packages of the previous VEO implementation. So, you can execute
+your program linked with the previous VEO implementation.
+
+
+### Installing development package
 
-veoffload-veorun is automatically installed by dependency.
+To develop programs, @PACKAGE@-devel and @PACKAGE@run-devel and
+thedevelopment packages of the compiler (2.2.2 or later) are also
```
Comment (efocht) [inline doc/GettingStarted.md.in]: typo, space missing

File: scripts/relink_aveorun.in [hunk 4]
```diff
@@ -39,7 +39,7 @@ while [ $# -gt 0 ]; do
       VERBOSE=-v
       ;;
     '-fopenmp')
-        default_LDFLAGS="${default_LDFLAGS} ${option} @velibdir@/liburpcVE_omp.a"
+        default_LDFLAGS="${default_LDFLAGS} ${option} @velibdir@/liburpcVE.a"
```
Comment (efocht) [inline scripts/relink_aveorun.in]: I don't understand this. When -fopenmp is selected in relink_aveorun, we want openmp!

File: doc/Restriction.md (hunk 1 above)
Comment (veos-sxarr) [reply doc/Restriction.md]: We pushed the patch to fix language.

File: src/Makefile (hunk 2 above)
Comment (veos-sxarr) [reply src/Makefile]: We pushed the patch to add relink_aveorun to aveorun package.
We made mk_veorun_static a symbolic link.

File: doc/GettingStarted.md.in (hunk 3 above)
Comment (veos-sxarr) [reply doc/GettingStarted.md.in]: Typo was already fixed in commit :4497f91.

https://github.com/veos-sxarr-NEC/aveo/commit/4497f91f4063eacb2b4bc22382490f3d64caf0de

**Review by efocht — APPROVED** [review]

File: scripts/relink_aveorun.in (hunk 4 above)
Comment (veos-sxarr) [reply scripts/relink_aveorun.in]: We already reverted this change.

https://github.com/SX-Aurora/aveo/pull/5/commits/48267ff3d55e1427ad27d4acba0fc16aa0fdfc3f

Currently, liburpcVE_omp.a is specified.

~~~
     '-fopenmp')
      default_LDFLAGS="${default_LDFLAGS} ${option} $VELIBDIR/liburpcVE_omp.a"
      is_openmp=1`
~~~

**Pull request merged**